jenkins/src/main/fabric8/jenkins-deployment.yml

```diff
@@ -107,6 +107,12 @@ spec:
         - mountPath: "/home/jenkins/.docker"
           name: "jenkins-docker-cfg"
           readOnly: false
+        - mountPath: /var/jenkins_home/jobs
+          name: jenkins-jobs
+          readOnly: false
+        - mountPath: /var/jenkins_home/workspace
+          name: jenkins-workspace
+          readOnly: false
       serviceAccountName: "jenkins"
       volumes:
       - hostPath:
```

Comment:
Once we have the S2I binary builds and ImageStreams working with Arquillian, Jenkins Pipelines and multiple environments; yes we'll be able to remove access to the docker socket on OpenShift. We'll probably need 2 apps really; the docker access flavour and the openshift-only-no-docker-socket flavour